Collaboration.md

```diff
@@ -1,8 +1,8 @@
 # Bitcoin Design Community collaboration playbook
 
-The goal of this document is to define common types of collaborations, to serve as starting points for new inquiries. Collaborations should be tracked as issues with the "collabration" label, see the [current ones here](https://github.com/BitcoinDesign/Meta/issues?q=is%3Aopen+is%3Aissue+label%3Acollaboration).
+The goal of this document is to define common types of collaborations, to serve as starting points for new inquiries. Collaborations should be tracked as issues with the "collaboration" label, see the [current ones here](https://github.com/BitcoinDesign/Meta/issues?q=is%3Aopen+is%3Aissue+label%3Acollaboration). Many collaborations we have had so far are shown on the [projects page](https://bitcoin.design/projects/).
 
-This is not meant to be a strict rule set. Every project is unique, and healthy collaboration requires everyone involved to be sensitive to the projects needs and goals. Our own goal is to support projects and contributors to improve their products and design practices (and not impose our ideas). 
+This is not meant to be a strict rule set. Every project is unique, and healthy collaboration requires everyone involved to be sensitive to the projects needs and goals. Our own goal is to support projects and contributors to improve their products and design practices (and not impose our ideas).
 
 Here are six types of design needs a project may have.
```

Comment:
```suggestion
Here are six types of ways a project can engage and collaborate with the community:
```

Reply:
Where does the seventh one come from? The page headers only go to `#6`.